src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Renamed.cs

```diff
@@ -150,7 +150,7 @@ public static void FileSystemWatcher_Moved_NestedDirectoryRootWithoutSubdirector
 
                 using (var dir2 = new TemporaryTestDirectory(Path.Combine(dir1.Path, "dir2")))
                 {
-                    Utility.ExpectNoEvent(createdOccured, "dir2 created", Utility.WaitForCreationTimeoutInMs);
+                    Utility.ExpectNoEvent(createdOccured, "dir2 created");
```

Comment:
It varies a ton...sometimes it took 300ms and sometimes it took 15 seconds; it was very strange. If a little amount of false negative flakiness is okay, then works for me :)